pages/cesium.js

```diff
@@ -142,7 +130,7 @@ var CesiumPreview = function() {
         resize();
 
         var gltfFileName = document.getElementById('gltfFileName').textContent;
-        var gltfRootPath = document.getElementById('gltfRootPath').textContent;
+        var gltfRootPath = "file://" + document.getElementById('gltfRootPath').textContent;
```

Comment:
I think technically there should be three slashes here, such that the "hostname" is blank.